content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -64,10 +65,12 @@ cluster for mesh expansion, run the following commands on a machine with cluster
     more about customizing Helm charts in the [Helm documentation](https://docs.helm.sh/using_helm/#using-helm).
     {{< /tip >}}
 
-1.  Find the IP address of the Istio ingress gateway, as this is how the mesh expansion machines will access [Citadel](/docs/concepts/security/) and [Pilot](/docs/concepts/traffic-management/#pilot-and-envoy).
+1.  Find the IP address of the Istio ingress gateway, as this is how the mesh expansion machines will access [Citadel](/docs/concepts/security/) and [Pilot](/docs/concepts/traffic-management/#pilot-and-envoy). VM workloads belong to certain namespace, which means their SPIFFE certificates and `ServiceEntry` should match to that namespace.
```

Comment:
If you're open to feedback on the text, I'd suggest splitting the second and third sentences here, plus the `export SERVICE_NAMESPACE` code bit, into a dedicated step since the namespace discussion has nothing to do with the gateway address. I think this will make it easier to a reader to understand.

Then update the namespace text to something like: In this step, we define the namespace to which the VM will join. Throughout this example, we'll reference the `SERVICE_NAMESPACE` variable; the value of this variable needs to match the namespace setting in configuration files that we later write.

Reply:
Thanks for suggestion! Applied.